lib/autobuild/packages/autotools.rb

```diff
@@ -32,8 +32,23 @@ class Autotools < Configurable
         attr_accessor   :autoheader_flags
         attr_accessor   :autoconf_flags
         attr_accessor   :automake_flags
+        attr_accessor   :bear_flags
 
         @builddir = 'build'
+        @@enable_bear_globally = false
+
+        def self.enable_bear_globally?
+            @@enable_bear_globally ? true : false
```

Comment:
[I thought we were avoiding the double negation idiom to make rubocop happy... ](https://github.com/rock-core/autoproj/blob/d264cbc424d6a656cd4466b5562b54796423675b/lib/autoproj/cli/base.rb#L211)

Reply:
In the case of the code you're linking to, the result needs to be "not nil". If rubocop does not like !!, it's because the base cop setup says that you simply should not convert to booleans at all (i.e. `@@enable_bear_globally` is enough (I forgot about the cop thingy)